photon-server/build.gradle

```diff
@@ -49,7 +53,25 @@ dependencies {
     implementation "org.apache.commons:commons-collections4:4.4"
     implementation "org.apache.commons:commons-exec:1.3"
 
-    // wpilib stuff
+    // JOGL stuff
+    implementation "org.jogamp.gluegen:gluegen-rt:$joglVersion"
+    implementation "org.jogamp.jogl:jogl-all:$joglVersion"
+
+    implementation "org.jogamp.gluegen:gluegen-rt:$joglVersion:natives-linux-amd64"
+    // implementation "org.jogamp.gluegen:gluegen-rt:$joglVersion:natives-linux-armv6"
+    implementation "org.jogamp.gluegen:gluegen-rt:$joglVersion:natives-linux-armv6hf"
+    implementation "org.jogamp.gluegen:gluegen-rt:$joglVersion:natives-linux-aarch64"
+    implementation "org.jogamp.gluegen:gluegen-rt:$joglVersion:natives-macosx-universal"
+    implementation "org.jogamp.gluegen:gluegen-rt:$joglVersion:natives-windows-amd64"
+
+    implementation "org.jogamp.jogl:jogl-all:$joglVersion:natives-linux-amd64"
+    // implementation "org.jogamp.jogl:jogl-all:$joglVersion:natives-linux-armv6"
+    implementation "org.jogamp.jogl:jogl-all:$joglVersion:natives-linux-armv6hf"
+    implementation "org.jogamp.jogl:jogl-all:$joglVersion:natives-linux-aarch64"
+    implementation "org.jogamp.jogl:jogl-all:$joglVersion:natives-macosx-universal"
+    implementation "org.jogamp.jogl:jogl-all:$joglVersion:natives-windows-amd64"
+
```

Comment:
Do we need the JOGL natives for platforms other than the Pi?

Reply:
I commented the non-Pi 4 ones out for now... Is that alright? I can also remove them.